docs/pages/accounts/account-types.mdx

```diff
@@ -136,3 +136,9 @@ Access for members is managed through a role-based system. Users can have the _o
 | **Admin**     | Can control most settings on your account, including signing up for paid services, changing permissions of other users, and managing programmatic access. |
 | **Developer** | Can create new projects, make new builds, release updates, and manage credentials.                                                                        |
 | **Viewer**    | Can only view your projects through Expo Go but cannot modify your projects in any way.                                                                   |
+
+### Security activity
+
+Security activity is a list of changes that happened to an EAS user profile – your personal account. It includes, among others, change of password, email, and 2FA authentication setup.
```

Comment:
> Is Security activity only available to users who are subscribed to an EAS Plan (that is not on free plan)?

It's available to all users.

>We haven't used EAS user profile and introducing this new term would require refactoring this doc to explain it.

Oops, I will remove it.

> Also, what happens to users who are part of organization?

Nothing changes since it's about user profile which is used to access all user's accounts but I need to explain it without using the user profile term.